notes/stories/film/film-scifi-2010s.st.txt

```diff
@@ -1645,6 +1645,58 @@ human nature [It was human nature to romanticize war, according to the malevolen
 piloted creature-shaped vehicle [Deunan wielded a powerful humanoid battle bot from within.]
 
 
+movie: Lucy (2014)
+==================
+
+:: Title
+Lucy
+
+:: Description
+Lucy is a 2014 French science fiction action film written and directed by Luc
+Besson for his company EuropaCorp, and produced by his wife, Virginie
+Besson-Silla.
+
+Synopsis: A woman gains psychokinetic abilities when a nootropic, psychedelic
+drug is absorbed into her bloodstream.
+
+:: Date
+2014-07-25
+
+:: References
+https://en.wikipedia.org/wiki/Lucy_(2014_film)
+
+:: Ratings
+2 <paul>
+
+:: Major Themes
+organized crime [The ruthless Mr. Jang was the mastermind behind an international drug smuggling ring. The story follows Lucy as she becomes forcibly made to serve as a drug mule for Mr. Jang and his fellow criminals.]
+speculative ability [The story follows Lucy as she acquires a host of superpowers after an experimental, illicit drugs gets into her bloodstream. Her superpowers included, super intelligence, mind reading, telekinesis, an ability to manipulate radio and other EM waves, and even time travel.]
+human nature [A message of the film is that a loss of humanity comes with increased intelligence.]
+apotheosis [The story culminated with Lucy transcending space and time upon reaching 100% usage of her brain capacity, and becoming one with the universe or some such thing.]
```

Comment:
A slight broadening of our definition, but I guess it's fair. Might update our theme definition to fit.

Reply:
Erring on the side of caution here, I revised "apotheosis" to be the more general "what if I underwent a physical metamorphosis". I noticed we have some other cases of "what if I underwent a physical metamorphosis", like the case of John Doe in tng3x25.